recipe/meta.yaml

```
@@ -53,28 +56,25 @@ requirements:
     - snappy
   run_constrained:
     - arrow-cpp-proc * {{ build_ext }}
+    - cudatoolkit >=9.2         # [cuda_compiler_version not in (undefined, "None")]
 
 test:
   imports:
     - pyarrow
-    - pyarrow.dataset  # [not py==27]
-    - pyarrow.flight   # [not py==27]
-    - pyarrow.gandiva  # [(not py==27) and (not (aarch64 or ppc64le))]
+    - pyarrow.dataset
+    - pyarrow.flight
+    - pyarrow.gandiva  # [not (aarch64 or ppc64le)]
     - pyarrow.orc      # [unix]
     - pyarrow.parquet
     - pyarrow.plasma   # [unix]
     - pyarrow.fs
     - pyarrow._s3fs
     - pyarrow._hdfs
-    - pyarrow.cuda                                            # [cuda_compiler_version not in (undefined, "None")]
-  requires:                                                   # [cuda_compiler_version not in (undefined, "None")]
-    - pytest                                                  # [cuda_compiler_version not in (undefined, "None")]
-    - numba                                                   # [cuda_compiler_version not in (undefined, "None")]
-  source_files:                                               # [cuda_compiler_version not in (undefined, "None")]
-    - python/pyarrow/tests/                                   # [cuda_compiler_version not in (undefined, "None")]
-  commands:                                                   # [cuda_compiler_version not in (undefined, "None")]
-    - python python/pyarrow/tests/test_cuda.py                # [cuda_compiler_version not in (undefined, "None")]
-    - python python/pyarrow/tests/test_cuda_numba_interop.py  # [cuda_compiler_version not in (undefined, "None")]
+    # We can only test importing cuda package but cannot run when a
+    # CUDA device is not available, for instance, when building from a
+    # CI. That said, the pyarrow.cuda import test is carried out in
+    # the build script.
```

@pearu, I think the import itself is fine here, so no need to refer to the build script (where I have removed it for now)?

```suggestion
    # CUDA device is not available, for instance, when building from CI.
```

---

Yes, I agree, the comment is obsolete.